lib/view/about_us_screen.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:pslab/constants.dart';
+import 'package:pslab/view/widgets/build_contact_list.dart';
+import 'package:pslab/view/widgets/main_scaffold_widget.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class AboutUsScreen extends StatefulWidget {
+  const AboutUsScreen({super.key});
+
+  @override
+  State<StatefulWidget> createState() => _AboutUsScreenState();
+}
+
+class _AboutUsScreenState extends State<AboutUsScreen> {
+  String get iconAboutUs => 'assets/images/logo.png';
+  final List<Map<String, dynamic>> contactItems = [
+    {
+      'icon': const Icon(Icons.mail),
+      'title': connectWithUs[1],
+      'url': 'mailto:$mail'
+    },
+    {'icon': const Icon(Icons.link), 'title': connectWithUs[2], 'url': website},
+    {
+      'icon': const Icon(FontAwesomeIcons.github, size: 20),
+      'title': connectWithUs[3],
+      'url': github
+    },
+    {
+      'icon': const Icon(Icons.facebook_sharp),
+      'title': connectWithUs[4],
+      'url': facebook
+    },
+    {
+      'icon': const Icon(FontAwesomeIcons.xTwitter, size: 20),
+      'title': connectWithUs[5],
+      'url': x
+    },
+    {
+      'icon': const Icon(FontAwesomeIcons.youtube, size: 20),
+      'title': connectWithUs[6],
+      'url': youtube
+    },
+    {
+      'icon': const Icon(Icons.person),
+      'title': connectWithUs[7],
+      'url': developers
+    },
+  ];
```

Comment:
```suggestion
  final List<Map<String, dynamic>> contactItems = [
    {'icon': const Icon(Icons.mail), 'title': connectWithUs[1], 'url': 'mailto:$mail'},
    {'icon': const Icon(Icons.link), 'title': connectWithUs[2], 'url': website},
    {'icon': const Icon(FontAwesomeIcons.github, size: 20), 'title': connectWithUs[3], 'url': github},
    {'icon': const Icon(Icons.facebook_sharp), 'title': connectWithUs[4], 'url': facebook},
    {'icon': const Icon(FontAwesomeIcons.xTwitter, size: 20), 'title': connectWithUs[5], 'url': x},
    {'icon': const Icon(FontAwesomeIcons.youtube, size: 20), 'title': connectWithUs[6], 'url': youtube},
    {'icon': const Icon(Icons.person), 'title': connectWithUs[7], 'url': developers},
  ];
```

Reduces line count, easier to read through the file.

Reply:
@CloudyPadmal This change was made by the dart formatter. I initially used single lines for these. If we don’t do this then the Common Build CI fails, since formatted code is a requirement.

Reply:
I see, it clips at 80 characters. Let's ignore this, in that case.